diff --git a/node_modules/naive-ui/es/tabs/src/Tabs.js b/node_modules/naive-ui/es/tabs/src/Tabs.js
index 91b7f0d..8b20271 100644
--- a/node_modules/naive-ui/es/tabs/src/Tabs.js
+++ b/node_modules/naive-ui/es/tabs/src/Tabs.js
@@ -69,6 +69,17 @@ export default defineComponent({
             tabChangeIdRef.id = 0;
             updateCurrentBarStyle();
         });
+        watch(mergedValueRef, () => {
+            void nextTick(() => {
+                const tabEl = getCurrentEl();
+                const { value: xScrollInst } = xScrollInstRef;
+                xScrollInst?.scrollTo({
+                    left: tabEl.offsetLeft,
+                    top: 0,
+                    behavior: 'smooth'
+                });
+            });
+        });
         function getCurrentEl() {
             var _a;
             const { value } = mergedValueRef;
